### ISSUE TITLE
Disable updates by default on Izzyondroid

### ISSUE THREAD

**Issue by theamazing0** [T8RIN/ImageToolbox#479]
I downloaded the app via Izzyondroid and was prompted to install a beta update. The beta update isn't available on izzyondroid. Apps that use Izzyondroid repos should auto-update the app, so the app doesn't need to check for updates and potentially confuse the user.

**Comment by T8RIN** [comment]
> I downloaded the app via Izzyondroid and was prompted to install a beta update. The beta update isn't available on izzyondroid. Apps that use Izzyondroid repos should auto-update the app, so the app doesn't need to check for updates and potentially confuse the user.

You can disable update sheets or beta versions in settings, but unfortunately i can't get if you installed the app from izzy or GitHub, or something else, because it is just the same apks :(

**Comment by theamazing0** [comment]
Yeah, but beta updates should still probably be disabled by default.

**Comment by T8RIN** [comment]
> Yeah, but beta updates should still probably be disabled by default.
> 
> On October 11, 2023 12:40:47 AM EDT, Malik Mukhametzyanov ***@***.***> wrote:
> >> I downloaded the app via Izzyondroid and was prompted to install a beta update. The beta update isn't available on izzyondroid. Apps that use Izzyondroid repos should auto-update the app, so the app doesn't need to check for updates and potentially confuse the user.
> >
> >You can disable update sheets or beta versions in settings, but unfortunately i can't get if you installed the app from izzy or GitHub, or something else, because it is just the same apks :(
> >
> >-- 
> >Reply to this email directly or view it on GitHub:
> >https://github.com/T8RIN/ImageToolbox/issues/479#issuecomment-1756786207
> >You are receiving this because you authored the thread.
> >
> >Message ID: ***@***.***>

Okay, I'll consider your offer 👀
